bin/osg-notify

```diff
@@ -173,6 +191,15 @@ def main():
     with open(args.message, 'r') as fp:
         contents = fp.read()
 
+        # Replace smart quotes and em/en dashes
+        contents = replace_smart_quotes_and_dashes(contents)
```

Comment:
Note that this smart quote/dash conversion to ascii is done even if the option to explicitly allow non-ascii characters is specified

Reply:
that's the expected behavior. Form the ticket:https://opensciencegrid.atlassian.net/browse/SOFTWARE-3893
"Also add --force option that allows the user to override the aforementioned error but still replaces smart quotes/dashes"

Reply:
Ok!  (@brianhlin - i guess i'm surprised that even when explicitly allowing non-ascii characters, smartquotes still get zapped, but it sounds like this was intentional in any case.)